frontend/controller/namespace.js

```diff
@@ -9,6 +9,10 @@ sbp('sbp/selectors/register', {
     const cache = sbp('state/vuex/state').namespaceLookups
     return cache?.[name] ?? null
   },
+  'namespace/lookupReverseCached': (id: string) => {
+    const cache = sbp('state/vuex/state').reverseNamespaceLookups
+    return cache?.[id] ?? null
```

Comment:
This function is essentially a copy-paste of the function above, handling the possibility of `reverseNamespaceLookups` (or namespaceLookups) not being defined. That was because the side-effects in the identity contract call the `lookupCached` selector, and I thought that an incorrect state should not cause the side-effect to fail. The `??null` part also coerces `undefined` into null, so that the function returns only two types instead of three (`string`, `undefined`, `null`).

Reply:
OK, I can get with `?? null` but not `cache?.[id]`. That should just be `cache[id]`.